programs/ambassadors/ambassadors-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Can we change the json schema version to use the current 2020-12?

Reply:
I tried updating it but got this error while testing should i chnage the script and manually check for each entry in schema?
![Uploading Screenshot 2024-06-21 at 8.31.38 PM.png…]()

Reply:
@benjagm Sir , could please advice what should we do...

Reply:
@ramishj, The image you uploaded is not visible, could you try uploading again  please?
![image](https://github.com/json-schema-org/community/assets/63534268/310418e6-539f-49e4-88b9-c11838de9ad4)